bootable-jar/WFLY-13305_Bootable_Jar.adoc

```diff
@@ -0,0 +1,505 @@
+    <resolve-expressions>true</resolve-expressions>
+  </cli-session>
+</cli-sessions>
+```
+
+===== Logging configuration
+
+During packaging the maven plugin generates a logging.properties file that reflects the server logging configuration. This ensures that the bootable JAR 
+will boot with a logging configuration in sync with the logging subsystem.
+
+IMPORTANT: Using a log4j appender as a `custom-handler` in the logging subsystem is not supported with the bootable JAR.
+           This only applies to custom handlers defined on the root of the logging subsystem. Logging profiles and
+           log4j configuration files located in your deployment will still work as expected.
+
+You can workaround this by supplying your own `logging.properties` and defining the path in the `boot-logging-config` maven plugin's configuration property.
+
+===== User application
+
+The WAR or JAR main artifact the Maven project is building is deployed and zipped along with the server. 
+The plugin option context-root=true|false (true by default) allows to rename the war file to ROOT.war 
+(doesn’t apply to other packaging types). 
+
+NB: Application deployment doesn't rely on the deployment scanner. The scanner is not required, it is even suggested to exclude it, useless in a bootable JAR context.
+For an example of how to exclude the _deployment-scanner_ Galleon layer, you can check https://github.com/wildfly-extras/wildfly-jar-maven-plugin/blob/master/examples/jaxrs/pom.xml#L35[this example]
+
+In case the bootable is an “hollow JAR”, no deployment is present in the JAR.
+
+===== Hollow JAR
+
+No WAR/JAR/EAR file is copied into the server deployments. This is controlled by a plugin option.
+The hollow JAR doesn’t require the deployment-scanner to be provisioned in order for the deployment to be taken into account by the runtime.
+At runtime, if the option ```--deployment=<deployment>``` is passed, the deployment artifact is copied to content dir and the xml configuration is updated with the deployment. 
+As an alternative, WildFly CLI can be used to deploy an application inside an hollow jar.
+ 
+
+==== Execute the bootable JAR
+
+This is done thanks to the command: 
+
+```
+java -jar <JAR name>.jar [arguments]
+```
+
+The detailed execution steps are:
+
+* Unzip the server to a directory
+* Handle arguments
+* Start the server
+* Wait for server end 
+
+Alternatively the Maven plugin can be used to run/shutdown the server from Maven wildfly-jar:run|start|shutdown.
+
+==== Configure the Bootable JAR at runtime
+
+The bootable JAR fully relies on the server configuration capabilities. It is not expected to define a new way to configure the server.
+The arguments passed to the bootable JAR are composed of bootable specific arguments and server arguments.
+
+Calling  java <JVM and system properties> -jar <bootable JAR name>.jar --help dumps the available arguments.
+
+==== Log manager
```

Comment:
Can we merge this sentence into a single section relating to logging?

Reply:
Done